pagopa/io-app
# feat: [IOPLT-178] Persist experimental context value

## Short description
This PR implements the logic to store and get the value of the experimental toggle directly from AsyncStorage.

## List of changes proposed in this pull request
- Implements an hook to recover the value and edit the context from external library 

## How to test
Change the value and then reload app, the value should be kept



https://github.com/pagopa/io-app/assets/3959405/1022e3a8-c167-4f28-93ce-9ea470d3b126
